<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaoZuo_ApplyReserve</class>
 <widget class="QDialog" name="LaoZuo_ApplyReserve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>[test]LaborMiner</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 252, 235);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>901</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>預定列表</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="ReserveTable">
      <property name="styleSheet">
       <string notr="true">font: 700 10pt &quot;微軟正黑體&quot;;
background-color: rgb(231, 255, 241);</string>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>成功列表</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="SuccessTable">
      <property name="styleSheet">
       <string notr="true">font: 700 10pt &quot;微軟正黑體&quot;;
background-color: rgb(231, 255, 241);</string>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnClearSuccess">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清除成功列表</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_RemainTime">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="RemainTimeText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>下一次嘗試剩餘時間：{time}</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>520</y>
     <width>711</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="RemainTimeText_2">
      <property name="text">
       <string>紀錄</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textbox_Logger"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
